app/src/main/java/edu/mills/heartoftaiwanese/activity/BaseViewModel.kt

```diff
@@ -0,0 +1,14 @@
+package edu.mills.heartoftaiwanese.activity
+
+import android.content.Context
+import edu.mills.heartoftaiwanese.repository.TranslationRepository
+
+open class BaseViewModel {
```

Comment:
What is the purpose of `BaseViewModel`? This isn't criticism. I'm curious.

Reply:
It holds a reference to the repository so that it gets shared.